bootstrap/sql/com.mysql.cj.jdbc.Driver/v015__create_db_connection_info.sql

```diff
@@ -1,3 +1,27 @@
 -- column deleted not needed for entities that don't support soft delete
 ALTER TABLE query_entity DROP COLUMN deleted;
 ALTER TABLE event_subscription_entity DROP COLUMN deleted;
+
+-- queries to rename params to connectionOptions for trino
+UPDATE dbservice_entity
+SET json = JSON_SET(
+    JSON_REMOVE(json, '$.connection.config.connectionOptions'),
```

Comment:
btw is there anything that users need to put in connectionOptions regularly? If so, can we make that as standalone property?

Reply:
There are some other auth types in trino like certificates and kerberos auth do we want to add those? Although it might be a bigger change so separate issue might be better for this
More info -> https://github.com/trinodb/trino-python-client/blob/master/README.md